newIDE/app/resources/inAppTutorials/inAppTutorialShortHeaders.json

```diff
@@ -26,7 +26,24 @@
   {
     "id": "joystick",
     "contentUrl": "https://resources.gdevelop-app.com/in-app-tutorials/joystick.json",
-    "availableLocales": ["en", "fr", "es", "pt", "th", "ar", "sq"],
```

Comment:
Why change only this tutorial?

Reply:
I'm updating tutorials one by one, not finished 😇